resources/Shaders/ForwardPlus.frag.glsl

```diff
@@ -104,27 +115,31 @@ void main()
 	int start = int(LightGrids.Data[currentTile].Start);
 	int amount = int(LightGrids.Data[currentTile].Amount);
 	//for(int i = 0; i < 3; i++)
-	for(int i = start; i < start + amount; i++)
-	{
+	for(int i = start; i < start + amount; i++) {
 		int l = int(LightIndex[i]);
-
-		LightResult result = CalcPointLight(V * PointLights.List[l].Position, PointLights.List[l].Radius, PointLights.List[l].Color, PointLights.List[l].Intensity, viewVec, position, normal, PointLights.List[i].Falloff);
-
+		LightSource light = LightSources.List[l];
+		LightResult result;
+
+		if(light.Type == 1) { // point
```

Comment:
Det är en Enum i cpp filen, men den blir tills int när man skickar in den i shadern.